.github/workflows/maven-check-deps-build-publish.yml

```diff
@@ -212,3 +231,13 @@ jobs:
     secrets:
       NEXUS_USERNAME: ${{ secrets.NEXUS_USERNAME }}
       NEXUS_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
+
+  notify-ocd3:
+    needs: publish
+    if: ${{ inputs.notify-ocd3 }}
+    uses: mekomsolutions/shared-github-workflow/.github/workflows/ocd3-notify.yml@main
+    with:
+      webhook-url: ${{ inputs.ocd3-webhook-url }}
+    secrets:
+      OCD3_USERNAME: ${{ inputs.ocd3-username }}
+      OCD3_PASSWORD: ${{ inputs.ocd3-password }}
```

Comment:
Using `ocd3-username` and `ocd3-password` makes them optional unlike when you use them as secrets. The actual value of ocd3-username is not displayed or exposed in the workflow file. It's securely stored and managed by GitHub.